--- /usr/local/bin/pip-review	2016-01-25 20:58:23.000000000 +0000
+++ /usr/local/bin/pip-review-debian	2016-01-25 20:59:32.000000000 +0000
@@ -7,6 +7,7 @@
 import logging
 import sys
 import json
+import pkgutil
 try:
     import urllib2 as urllib_request  # Python2
 except ImportError:
@@ -167,11 +168,16 @@
             name = line.split('#egg=', 1)[1]
             if name.endswith('-dev'):
                 name = name[:-4]
-            yield name, 'dev', 'dev', True
+            #yield name, 'dev', 'dev', True
+            pkg_dir = pkgutil.get_loader(name.split('.')[0])
+            if pkg_dir is not None and pkg_dir.filename.startswith('/usr/local/lib/python'):
+                yield name, 'dev', 'dev', True
         else:
             name, version = line.split('==')
-            yield name, parse_version(version), version, False
-
+            #yield name, parse_version(version), version, False
+            pkg_dir = pkgutil.get_loader(name.split('.')[0])
+            if pkg_dir is not None and pkg_dir.filename.startswith('/usr/local/lib/python'):
+                yield name, parse_version(version), version, False
 
 class StdOutFilter(logging.Filter):
     def filter(self, record):
